test/package.js

```diff
@@ -0,0 +1,24 @@
+import fs from 'fs';
+import test from 'ava';
+import index from '../';
+
+test.cb('Every rule is defined in index file', t => {
+	fs.readdir('../rules/', function (err, files) {
+		t.ifError(err);
+
+		var rules = files.filter(file => file.indexOf('.js') === file.length - 3);
```

Comment:
Wondering about something... why is `esnext` not activated in XO for test files? You can activate rules for a glob right? Why not activate it for things in the default AVA test glob?